docs/plugins.md

```diff
@@ -748,6 +747,23 @@ Example of metric shape for object deletion:
 change_log{aggr="umeng_aff300_aggr2", cluster="umeng-aff300-01-02", datacenter="u2", index="2", instance="localhost:12993", job="prometheus", node="umeng-aff300-01", object="volume", op="delete", style="flexvol", svm="harvest", volume="harvest_demo"} 1698735708
 ```
 
+### Metric Value Changes
+
+When a metric value changes, the ChangeLog plugin will publish a metric with the following labels. Note that old and new values are not tracked to avoid impacting cardinality in Prometheus:
+
+| Label        | Description                                                                 |
+|--------------|-----------------------------------------------------------------------------|
+| object       | name of the ONTAP object that was changed                                   |
```

Comment:
How to enabled the changelog for metric ?
We may need to add example where to add `read_latency` if want to monitor that metric.

does adding `read_latency` here would work?
```
plugins:
  - ChangeLog:
      track:
        - aggr
        - node
        - state
```